src/styles/blocks/_day.scss

```diff
@@ -0,0 +1,22 @@
+.day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
+  width: $cell-width;
+  height: $cell-width;
+  background-color: #eee;
+  transition: 0.5s;
+
+  &:hover {
+    background-color: pink;
+    transform: translateY(-20px);
+    cursor: pointer;
+  }
+
+  @for $index from 1 through 31 {
+    &:nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+}
```

Comment:
Does the `day `make sense without a calendar? Better make it an `calendar`'s element